core/core_constants.cpp

```diff
@@ -710,7 +710,7 @@ void register_global_constants() {
 	BIND_CORE_BITFIELD_FLAG(PROPERTY_USAGE_EDITOR_BASIC_SETTING);
 	BIND_CORE_BITFIELD_FLAG(PROPERTY_USAGE_READ_ONLY);
 	BIND_CORE_BITFIELD_FLAG(PROPERTY_USAGE_SECRET);
-
+	BIND_CORE_BITFIELD_FLAG(PROPERTY_USAGE_SHOW_IF_ROOT);
 	BIND_CORE_BITFIELD_FLAG(PROPERTY_USAGE_DEFAULT);
```

Comment:
```suggestion

	BIND_CORE_BITFIELD_FLAG(PROPERTY_USAGE_DEFAULT);
```
Keep an empty line separating these for readability